src/AppBundle/Controller/Admin/UserController.php

```diff
@@ -38,8 +38,9 @@ public function viewAction(Request $request, User $user)
             new AdminUserType(),
             $user,
             [
-                'manager' => $this->getDoctrine()->getManager(),
-                'locale' => $request->getLocale(),
+                'manager'       => $this->getDoctrine()->getManager(),
+                'locale'        => $request->getLocale(),
+                'wantToLearn'   => (int)$user->getWantToLearn()
```

Comment:
Yes, and having a boolean variable to have the values true, false, 1, 0 or null was not readable. Looks much better now.